lib/ansible/runner/credential/ibmcloud_credential.rb

```diff
@@ -0,0 +1,16 @@
+module Ansible
+  class Runner
+    class IbmCloudCredential < Credential
+      def self.auth_type
+        "ManageIQ::Providers::IbmCloud::IbmCloudCredential"
```

Comment:
You might want to make this more Power specific, subclass under `ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager` perhaps

Reply:
Can this class live in the provider plugin?

Reply:
@Fryguy I need to check if it'd work that way. If so, then I'd gladly move it into our own plugin.

Reply:
This credential will be used to SSH into a PowerVC provider right? I'm wondering if something like `PowerVCCredential` or maybe `ImageImportSSHCredential` would be better?

Reply:
@jaywcarman Hi. No, it's not the SSH credential but the IBM Cloud credential for access to the Cloud Object Storage from the powervc server. The SSH credential is a separate issue which I'll handle in similar way. But you have a point, I should probably change the name to something that better reflects it's nature.